services/consul/Dockerfile

```diff
@@ -21,9 +21,3 @@ rm consul_${CONSUL_VERSION}_web_ui.zip
 # Expose standard ports, as well as ports used by consul
 EXPOSE 8300 8301 8301/udp 8302 8302/udp 8400 8500 53/udp
 
-ADD ./config /consul/config/
-ADD startconsulserver.sh .
-ADD startconsulclient.sh .
-
```

Comment:
With ansible, the commands are executed directly, these scripts are not used anymore. Config is loaded via volume instead of hardwired at image creation. This might also flatten the way to use the official consul image instead of our own one.